000-index.md

```diff
@@ -1,6 +1,7 @@
 # Node.js Enhancement Proposals
 
-* [Public C++ Streams](001-public-stream-base.md)
-* [ES6 Module Interoperability](002-es6-modules.md)
-* [WHATWG URL API](003-url.md)
-* [Port core to uv_link_t](004-uv_link_t.md)
+* 001 - [Public C++ Streams](001-public-stream-base.md) - REJECTED
+* 002 - [ES6 Module Interoperability](002-es6-modules.md) - DRAFT
+* 003 - [WHATWG URL API](003-url.md) - DRAFT
+* 004 - [Port core to uv_link_t](004-uv_link_t.md) - DRAFT
```

Comment:
I checked the status of each of them when I submitted the PR.  Will check again.  If they are right I'll go ahead and land.